ibis/backends/polars/compiler.py

```diff
@@ -87,25 +87,25 @@ def literal(op, **_):
     value = op.value
     dtype = op.dtype
 
-    if dtype.is_array():
-        value = pl.Series("", value)
-        typ = PolarsType.from_ibis(dtype)
-        val = pl.lit(value, dtype=typ)
-        return val.implode()
+    if dtype.is_interval():
+        return _make_duration(value, dtype)
```

Comment:
have to move this up here because there are some interval dtypes that _make_duration() can handle, but `PolarsType.from_ibis(dtype)` errors on.